python/tests/catalog/test_base.py

```diff
@@ -157,7 +158,7 @@ def update_namespace_properties(
 TEST_TABLE_IDENTIFIER = ("com", "organization", "department", "my_table")
 TEST_TABLE_NAMESPACE = ("com", "organization", "department")
 TEST_TABLE_NAME = "my_table"
-TEST_TABLE_SCHEMA = Schema(schema_id=1)
+TEST_TABLE_SCHEMA = Schema(NestedField(1, "foo", StringType(), True), schema_id=1)
```

Comment:
Is it still okay to have an empty table schema?

Reply:
This was during testing. This wasn't possible for some time because of all the changes I made to the code, but since this is initialized before the test, it would crash right away before hitting any breakpoints. I'll revert this.